xdsl/dialects/arm_neon.py

```diff
@@ -179,34 +181,40 @@ def assembly_line(self):
 
 
 @irdl_op_definition
-class DSSFMulVecScalarOp(ARMInstruction):
+class DSSFMulOp(ARMInstruction):
     """
-    Floating-point multiply (mixed: first source operand is a vector, second is a scalar. Destination is a vector)
-    This instruction multiplies each of the floating-point values in the first source operand by the
-    second source operand and writes the resulting values to the corresponding lanes of the destination.
+    Floating-point multiply. Different instruction types supported:
+
+    1. Vector:  multiplies corresponding floating-point values in the vectors in the two
+    source NEON registers, and writes the result vector to the destination.
+
+    See external [documentation](https://developer.arm.com/documentation/100069/0606/SIMD-Vector-Instructions/FMUL--vector-).
```

Comment:
Points to an old version. Is this intended?

Reply:
it's the lateset version I can find online. the docs for neon are hard to navigate but as far as I can tell if you try to switch it to the latest version on the menu you just get a asm manual

Reply:
I see